packages/agreement-process/src/services/agreementService.ts

```diff
@@ -412,7 +412,6 @@ export function agreementServiceBuilder(
 
       const agreements = (
         await readModelService.getAllAgreements({
-          producerId: agreement.data.producerId,
```

Comment:
Unrelated, but reviewing the submission logic I found this difference with the Scala code